### PR TITLE
[Fix] Gifs in Gallery

### PR DESCRIPTION
**Pull request by Mikunj**
- This will show a 'GIF' overlay on all gif images, that originate from Kitsu, in the image grid.
    - This only applies to grids which have more than 1 image.

simulator screen shot - iphone 6 - 2018-06-28 at 13 18 56

### CODE REVIEW

File: src/common/utils/url.js [hunk 1]
```diff
@@ -10,15 +10,13 @@ import { isEmpty } from 'lodash';
  * @returns Whether a url is a kitsu url.
  */
 export function isKitsuUrl(url) {
-  if (isEmpty(url)) return false;
-
   // Parse it
-  const parsed = parseURL(url);
+  const parsed = parseURL((url || ''));
```
Comment (wopian) [inline src/common/utils/url.js]: You can do the following instead of `||` everywhere:
```js
export function isKitsuUrl(url = '') {
```

Comment (Mikunj) [reply src/common/utils/url.js]: Ah thanks, not used to JS still, i'll quickly commit the changes.

Comment (Mikunj) [reply src/common/utils/url.js]: A quick question, does doing the change above protect the url from being `null` or `undefined`?

I tried this in REPL
```
function isUrl(url = 'default') {
  console.log(url);
}

isUrl();
isUrl('abc');
isUrl(null);
```
and it outputted
```
default
abc
null
```

Comment (wopian) [reply src/common/utils/url.js]: Oh I guess the current `|| ''` is more ideal in this case.

**Pull request merged**